apps/notifications/src/app/note/style.scss

```diff
@@ -157,3 +165,40 @@
 		text-decoration: none;
 	}
 }
+
+.wpnc__user {
+	.wpnc__note & {
+		padding: 0;
+	}
+
+	.wpnc__user-title {
+		font-weight: 500;
+	}
+}
+
+/**
+ * Comment
+ */
+.wpnc__note.wpnc__comment {
+	.wpnc__user {
+		padding: 16px;
+	}
+}
+
+/**
+ * Likes
+ */
+.wpnc__note.wpnc__like {
+	.wpnc__user {
+		padding-bottom: 16px;
+		border-bottom: 1px solid $gray-300;
+
+		&:not(:first-child) {
+			padding-top: 16px;
+		}
+	}
+
+	.wpnc__paragraph {
+		padding: 16px 0 0 !important;
```

Comment:
Using `!important` as a workaround. Will remove it when we get rid of the v1 styles...